File: Algorithms/Challenges/Arrays/OddCountElements/Python/OddCountElementsTest.py
import unittest
import OddCountElements as oce

class OddCountElementsTest(unittest.TestCase):

    def test_equal(self):
        arr = [ 5, 1, 2, 4, 4, 3, 4, 1, 5 ]
        result = [ 2, 3, 4 ]

        # assertEquals(expected, actual, delta);
        self.assertEquals(result, oce.oddTimeElems(arr));

if __name__ == '__main__':
    unittest.main()

File: Algorithms/Challenges/Arrays/OddCountElements/Java/OddCountElementsTest.java
import org.junit.Test;
import static org.junit.Assert.*;

public class OddCountElementsTest {
  private static final double DELTA = 1e-15;

  @Test
  public void exampleTest() {
    int[] arr = { 5, 1, 2, 4, 4, 3, 4, 1, 5 };
    int[] result = { 2, 3, 4 };

    OddCountElements oce = new OddCountElements();

    // assertEquals(expected, actual, delta);
    assertArrayEquals(result, oce.oddTimeElems(arr));
  }
}
